modules/appeals_api/app/models/concerns/appeals_api/appeal_scopes.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module AppealScopes
```

Comment:
I made a new concern for these scopes since they are no different from appeal to appeal. I think a lot more of our appeal model code could be extracted into concerns, but I only included the minimum here to keep things simple.